.github/workflows/buildmgr.yml

```
@@ -165,7 +166,9 @@ jobs:
         run: |
           VERSION=$(git show-ref ${{ github.ref }} | awk -F/ '{print $NF}')
           rm -rf buildmgr/${VERSION}
+          rm -f buildmgr/latest
           mkdir -p buildmgr/${VERSION}
+          ln -s ${VERSION}/ buildmgr/latest
```

I guess it should be:
```suggestion
          ln -s buildmgr/${VERSION}/ buildmgr/latest
```

---

I think not, the target of a symlink needs to be relative to the link, not tu the current working directory. Please could you double check that e.g., in your local Linux env?

---

I have double checked, you are absolutely right.